Monika After Story/game/script-topics.rpy

```diff
@@ -13244,3 +13244,29 @@ label monika_songwriting:
     $ _if = "when" if mas_isMoniEnamored(higher=True) else "if"
     m 1eua "I'd love to sing with you [_if] I come to your world, [player]."
     return
+
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_sweatercurse",category=['advice'],prompt="Sweater curse",random=True))
+
+label monika_sweatercurse:
+    m 1tub "Have you ever heard of {i}the curse of the love sweater?{/i}"
+    m 1hub "Ahaha! What a weird name, right?"
+    m 3eub "But it's actually an interesting superstition--and one that might actually have a cause!"
+    m 3eua "The 'curse,' or so it's named, states that if someone gives a hand-knit sweater to their romantic partner..."
+    m 3eka "It will lead to the couple breaking up."
+    m 2lsc "You might think that a gift that requires so much work and investment would have the {i}opposite{/i} effect..."
+    m 2esd "But there are actually a few logical reasons why this curse might exist."
+    m 4esd "Firstly... well, knitting a sweater just takes a {i}lot{/i} of time. It could take around a year!"
+    m 4eka "Over all those months, something bad might happen that causes the couple to fight and eventually separate."
+    m 4ekc "Or worse--the knitter might be trying to make the sweater as a great gift to save an already suffering relationship."
+    m 1rkd "There's also the likely possibility that the recipient... doesn't like the sweater that much."
+    m 1tkd "After they put so much money and time into knitting it, imagining their partner happily wearing it... it would really hurt to see it cast aside."
```

Comment:
I think swapping "money and time" around to "time and money" flows more nicely and sounds more natural.